config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
```

Comment:
Fixed in https://github.com/openaustralia/planningalerts-app/commit/472d0886ff25c42affd41cdfe58c35d7ca6e14bc